scripts/local-deploy.sol

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import "forge-std/Script.sol";
+import "../src/CosmoShips.sol";
+import "../src/GameLeague.sol";
+import "../src/IAttributeVerifier.sol";
+import "../test/fixtures/mockVerifier.sol";
+import "../test/fixtures/mockRandomGenerator.sol";
+
+contract MyScript is Script {
+    function run() external {
+        uint256 mintPrice = 1_000_000_000_000;
+        uint256 deployerPrivateKey = vm.envUint("LOCAL_PRIVATE_KEY");
```

Comment:
Not required

Reply:
Not sure about that. Please look at [this](https://book.getfoundry.sh/tutorials/solidity-scripting) example.
Why is it mentioned?

Reply:
resolved